### DISCUSSION TITLE
Saleor 3.3 Release

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#9648] [Announcements]
Hi Community! Saleor 3.3 has arrived, bringing with it full-text search for products, a new API for fetching metadata, and a lot more! Check the full changelog here: https://github.com/saleor/saleor/releases/tag/3.3.0

**Comment by patrys** [comment]
Version 3.3.1 fixes two minor issues you may have faced during migration to 3.3.0.

**Comment by jbergstroem** [comment]
For those of you reading this, you want to be using 3.3.2!
